Operational-Docs/scrape-job-instance-naming.md

```diff
@@ -0,0 +1,29 @@
+# How to create better job instance names through fqdn-based scraping
```

Comment:
Maybe:

# How to have human readable prometheus job instance names in Grafana dashboard

Reply:
I prefer "ensure", other than this - very well.